pkgs/development/python-modules/suds/default.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  isPyPy,
+  pytestCheckHook,
+  python311Packages,
+}:
+
+buildPythonPackage rec {
+  pname = "suds";
+  version = "1.1.2";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-HVz6dBFxk7JEpCM/JGxIPZ9BGYtEjF8UqLrRHE9knys=";
+  };
+
+  propagatedBuildInputs = with python311Packages; [
+    six
+  ];
```

Comment:
```suggestion
  dependencies = [ six ];
```

Reply:
and `six` should go to `nativeBuildInputs` because it isn't needed in runtime.